bluejeans/src/bluejeans_create.js

```diff
@@ -58,8 +68,8 @@ async function _command(params = {}, commandText, secrets = {}) {
     title: title,
     description: desc ? desc : '',
     timezone: 'America/New_York',
```

Comment:
Is there a scope to specify the timezone to which the meeting can be scheduled?

Reply:
The `timezone` field has no effect. It's just there because they made it a requirement. And to create meeting according to specific time, we can use`GMT+`. Something like this: `-start "03/05/20 GMT+0530"`. I know it's not so human friendly but we might need a lib to support normal codes. I don't know what codes the `Date` supports but `-start "03/05/20 19:20 PST"` works.

Reply:
Does the API document this? Would be nice to know the timezones allowed.

Reply:
I couldn't find the list of shortcodes allowed in `Date` in JS.